src/views/Home/components/DiningBalance/index.jsx

```diff
@@ -1,4 +1,14 @@
-import { Button, Card, CardContent, CardHeader, Grid, Link, Typography } from '@mui/material';
+import {
```

Comment:
Why is this import statement expanded across multiple lines?

Reply:
It is automatically with the Prettier extension we have been using